packages/fsengagement/src/GestureHandler.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import {
+  PanResponder,
+  View
+} from 'react-native';
+
+export interface GestureHandlerProps {
+  onSwipe: () => void;
+  setScrollEnabled: (enabled: boolean) => void;
+}
+export interface GestureHandlerState {
+  stateSet: boolean;
+}
+export interface GestureState {
+  dy: number;
+  dx: number;
+}
+export default class GestureHandler extends Component<GestureHandlerProps, GestureHandlerState> {
+  _panResponder: any;
+  constructor(props: GestureHandlerProps) {
+    super(props);
+    this.state = {
+      stateSet: false
+    };
+  }
+
+  componentWillMount(): void {
```

Comment:
`componentWillMount` is deprecated. You can probably move this to either the constructor or `componentDidMount`.